packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -352,6 +352,98 @@ describe('plugin-meetings', () => {
         });
       });
 
+      describe('virtual background effect', () => {
```

Comment:
none of the tests check that the right constructor was called

Reply:
In these tests we're not stubbing the constructor and using the real one so without stub we can't test whether it's called or not. Do we want me to mock the effects constructor and then test it?

Reply:
Added the `instanceOf` check in test cases.